front-end/src/App.js

```diff
@@ -7,6 +7,8 @@ import Home from "./pages/Home";
 import Layout from "./pages/Layout";
 import RegisterCourse from "./pages/RegisterCourse";
 import UserInformationScreen from "./pages/UserInformationScreen";
+import Login from "./components/Login";
+import LandingPage from "./components/LandingPage";
```

Comment:
I think that the imports of Login and LandingPage should also be removed since not being used